docs/CHANGELOG.asciidoc

```diff
@@ -72,6 +72,432 @@ Review important information about the {kib} 7.16.x releases.
+
+For the Elastic Security 7.16.0 deprecations, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+[discrete]
+[[deprecation-118363]]
+.Adds levels to rule registry deprecations
+[%collapsible]
+====
+*Details* +
+Levels are now explicitly in deprecation configurations. For more information, refer to {kibana-pull}118363[#118363].
+
+*Impact* +
+Levels for deprecations are explicitly set.
+====
+      
+[discrete]
+[[deprecation-114578]]
+.Deprecates `search:includeFrozen`
+[%collapsible]
+====
+*Details* +
+The <<search-includefrozen,`search:includeFrozen`>> advanced setting is deprecated and will be removed in 9.0.0. For more information, refer to {kibana-pull}114578[#114578].
+
+*Impact* +
+In 7.16.0, there is no impact.
+====
+      
+[discrete]
+[[deprecation-114381]]
+.Sets `ignore_throttled` when `search:includeFrozen` is `true`
+[%collapsible]
+====
+*Details* +
+When the <<search-includefrozen,`search:includeFrozen`>> advanced setting is set to `true`, `ignore_throttled` is conditionally configured. For more information, refer to {kibana-pull}114381[#114381].
+
+*Impact* +
+When you open *Upgrade Assistant*, then click *View deprecation logs in Observability*, `ignore_throttled` deprecation logs appear.
+====
+      
+[discrete]
+[[deprecation-114019]]
+.Deprecates `xpack.reporting.capture.viewport`
+[%collapsible]
+====
+*Details* +
+The `xpack.reporting.capture.viewport` settings are deprecated and will be removed in 8.0.0. For more information, refer to {kibana-pull}114019[#114019].
+
+*Impact* +
+When you upgrade to 8.0.0, you are unable to use the `xpack.reporting.capture.viewport` settings.
+====
+      
+[discrete]
+[[deprecation-113364]]
+.Removes the `apm_oss plugin` and deprecates related settings
+[%collapsible]
+====
+*Details* +
+The `apm_oss` plugin has been removed as a dependency of the `apm` plugin, and the `apm_oss` settings have been deprecated and moved to the `xpack.apm.indices*` configuration path. For more information, refer to {kibana-pull}113364[#113364].
+
+*Impact* +
+Update the following settings:
+
+* `apm_oss.errorIndices` is deprecated. Use `xpack.apm.indices.error`.
+* `apm_oss.metricsIndices` is deprecated. Use `xpack.apm.indices.metric`.
+* `apm_oss.onboardingIndices` is deprecated. Use `xpack.apm.indices.onboarding`.
+* `apm_oss.sourcemapIndices` is deprecated. Use `xpack.apm.indices.sourcemap`.
+* `apm_oss.spanIndices` is deprecated. Use `xpack.apm.indices.span`.
+* `apm_oss.transactionIndices` is deprecated. Use `xpack.apm.indices.transaction`.
+====
+      
+[discrete]
+[[deprecation-111676]]
+.Deprecates `xpack.security.enabled`
+[%collapsible]
+====
+*Details* +
+The `xpack.security.enabled` setting is deprecated and will be removed in 8.0.0. For more information, refer to {kibana-pull}111676[#111676].
+
+*Impact* +
+Use {ref}/security-settings.html#general-security-settings[`xpack.security.enabled`] in elasticsearch.yml.
+====
```

Comment:
This was already deprecated in 7.11: https://www.elastic.co/guide/en/kibana/7.11/release-notes-7.11.0.html

However, we did deprecate disabling most _other_ plugins in the 7.16 release: #89584

Reply:
I tried to clear this up, but any other suggestions are appreciated.

Reply:
I guess I would suggest just removing this section.